diff --git a/op_builder/gds.py b/op_builder/gds.py
index e024674e..c9b67a7b 100644
--- a/op_builder/gds.py
+++ b/op_builder/gds.py
@@ -36,6 +36,10 @@ class GDSBuilder(AsyncIOBuilder):
         return super().extra_ldflags() + ['-lcufile']

     def is_compatible(self, verbose=False):
+        if self.is_rocm_pytorch():
+            if verbose:
+                self.warning(f'{self.NAME} is not compatible with ROCM')
+            return False
         import torch.utils.cpp_extension
         CUDA_HOME = torch.utils.cpp_extension.CUDA_HOME
         CUDA_LIB64 = os.path.join(CUDA_HOME, "lib64")
